src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -280,16 +280,29 @@ public void AddFile(Dictionary<string, bool> setup, string thisFile)
                 if (string.IsNullOrWhiteSpace(line) || line.StartsWith("//"))
                 {
                     continue;
-                }
-
+                }
+
+                // .Net5: indicates expected output expression for .Net 5 and above
+                if (line.StartsWith(".Net5:"))

```

Comment:
>if (line.StartsWith(".Net5:")) [](http://example.com/codeflow?start=16&length=30)

Per comment in the .txt files, can we avoid this and absorb the difference in the runtime functions?